pkg/cmd/pr/view/view.go

```diff
@@ -101,6 +103,14 @@ func viewRun(opts *ViewOptions) error {
 		return utils.OpenInBrowser(openURL)
 	}
 
+	opts.IO.StartProgressIndicator()
```

Comment:
This starts the progress indicator after it had just finished around `PRFromArgs`. Would we consider letting it spin until all the necessary data has been fetched?

Reply:
I started out this direction but the code ended up pretty messy with the extra conditional checks. When I tried out the current solution the UI experience was exactly the same since the time between the starts and stops was imperceptible making it seem like it never left the screen. I don't have a strong opinion though.